spdk.spec

```diff
@@ -198,6 +198,14 @@ cp build/examples/nvme_manage %{buildroot}/%{_bindir}/spdk_nvme_manage
 # Change /usr/bin/{env ,}bash to resolve env-script-interpreter rpmlint error.
 sed -i -e '1s/env //' %{install_datadir}/scripts/setup.sh
 
+# Install rpc.py tool
+cp scripts/rpc.py %{install_datadir}/scripts/
+mkdir -p %{install_datadir}/scripts/rpc
+cp scripts/rpc/*.py %{install_datadir}/scripts/rpc/
```

Comment:
This same logic/process needs applying to the debian package also.  Please see https://github.com/daos-stack/spdk/blob/2d53afdbf7f52775d27c12649f212169e5e0ba2d/debian/rules#L47-L48 for where you need to add them.  But I am also noticing that the operations on line 190-196 are not there either.  Can you please rectify that in this PR also so that we are doing the same thing for debian packages as we are doing for the RPMs?

But for this file, let's move the above lines above line 199 and then just add the `rpc.py` filename to the `sed` on line 199.

Reply:
Done for `spdk.spec`

Reply:
@brianjmurrell do you mean changes introduced by #58?

Reply:
Heh.  Yes, more or less.  Or perhaps exactly.  :-)  Please feel free to import that into your PR in whatever form you wish.  The ultimately goal of course is to have the debian package be equivalent to the RPM.

Reply:
Done